### ISSUE TITLE
Add a Pa11y cookbook

### ISSUE THREAD

**Issue by hollsk** [pa11y/pa11y.github.io#35]
Perhaps a cookbook for common (or even relatively exotic) tasks would help some users? I'm thinking something like http://restcookbook.com/ or http://deployer.org/recipes - not quite a user guide, but not quite a bunch of how-to tutorials either.

**Comment by rowanmanning** [comment]
That's a great idea 👍

**Comment by j-mes** [comment]
That actually would help me with sorting specific needs when I would be using Pa11y :+1:

**Comment by arndissler** [comment]
Isn't this covered by the `tutorials.yml`?

**Comment by josebolos** [comment]
I agree this is better covered by the tutorials. Due to our lack of resources, I think I rather focus on improving the documentation if something is not clear, than try to write guides for specific issues.
